test/unit/core/type_traits/range_iterator_test.cpp

```diff
@@ -72,20 +72,18 @@ TEST(range_and_iterator, value_type_)
     using foreign_iterator = seqan3::detail::random_access_iterator<std::vector<int>>;
     auto v = std::views::iota(1);
 
-    using type_list_example = seqan3::type_list<seqan3::value_type_t<std::vector<int>>, // short
-                                                typename seqan3::value_type<std::vector<int>>::type, // long
```

Comment:
For this expression

```c++
typename seqan3::value_type<std::vector<int>>::type, // long
```

there is no equivalent in the standard, because they only support the short notation `seqan3::value_type_t`.

See https://en.cppreference.com/w/cpp/ranges/iterator_t

So we just remove this test case.

(see https://github.com/seqan/seqan3/pull/1700#discussion_r402271060)